src/pages/result/SurveyIdLoaded.tsx

```diff
@@ -44,6 +47,8 @@ const PILL_COLORS: Color[] = ['bluegreen', 'pink', 'skyblue', 'yellowgreen', 'pu
 
 const SurveyIdLoaded = ({ surveyId }: Props) => {
   const { fireToast } = useToast();
+  const [isRevisit, setIsRevisit] = useLocalStorage<boolean>(`${LOCAL_STORAGE_KEY.resultRevisit}`, false);
```

Comment:
```suggestion
  const [isRevisit, setIsRevisit] = useLocalStorage<boolean>(LOCAL_STORAGE_KEY.resultRevisit, false);
```

상수가 문자열이라 이렇게 써도 되지 않을까욘?? 🤔

Reply:
오옹 이렇게 쓰면 어떤 변수가 들어와도 문자열임을 보장할 수 있어서 권장하는 방법인가요?! 궁금 @hyesungoh

Reply:
```ts
`${LOCAL_STORAGE_KEY.resultRevisit}`
```

산됴우님이 작성하신 방법은 말씀하신 것처럼 어떤 값이 들어와도 문자열임을 보장할 수 있는 방법인 거 같아용

근데 제가 말씀드린

```ts
LOCAL_STORAGE_KEY.resultRevisit
```

요건 저장된 값이 문자열이니, 그냥 사용해도 되지 않을까 ~~~ 하는 생각입니다 !

Reply:
아아 반대로 봤네용! 굿굿 ㅎㅎ

Reply:
[c9b4833](https://github.com/depromeet/na-lab-client/pull/458/commits/c9b4833c6109bfcb5af5759d25c79334adbc3fc7)에서 수정 완~~~!